core-web/package.json

```diff
@@ -112,6 +112,7 @@
     "react-dom": "^18.2.0",
     "regenerator-runtime": "^0.13.9",
     "rxjs": "~6.6.3",
+    "superstruct": "^1.0.3",
```

Comment:
Did you check the final file size after adding this? nothing to worry about?

Reply:
Size information for superstruct: https://bundlephobia.com/package/superstruct@1.0.3

It was  3.05MB -> 3.12 MB